content/en/docs/Getting started/checking-out-the-source-code.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Checking out the source code"
 linkTitle: "Checking out the source code"
-weight: 1
+weight: 4
```

Comment:
What does this `weight` property represent? It seems to align with the list in `_index.md` so I'm assuming it is related to navigation somehow?

Reply:
Thanks for the review Brian. The value of [weight determines the order](https://gohugo.io/templates/lists/#by-weight) of a list of links in dynamically generated navigation menus. In this case, I'm ordering the left-hand 'Getting started' menu according to the sequence a group of tasks need to be completed rather than the default alphabetical ordering.